src/coreclr/vm/stublink.cpp

```diff
@@ -1815,6 +1815,7 @@ bool StubLinker::EmitUnwindInfo(Stub* pStubRX, Stub* pStubRW, int globalsize, Lo
 
 #ifdef TARGET_AMD64
     // Publish Unwind info to ETW stack crawler
+    UnwindInfoTable::PublishUnwindInfo(false);
```

Comment:
Should we limit this to the interpreter scenario or is this general goodness for the runtime?

Reply:
Unwinding through manually emitted stubs has always been a bug farm. It turns out we have not had any manually emitted stubs that require unwinding for a while. I am deleting the complicated buggy infrastructure in #108684. (I was looking at some obviously buggy code related to the StubLinker unwinding when reviewing #107282 only to find out that it is not actually used and that's why we never seen the bug... .)

This interpreter stub is the last outlier. I think that this fix is fine to help the experiment to move along. Is your goal still "proof of concept and plan for getting rid of all dynamically generated code with interpreter"? It means that you will need to figure out how to get rid of the interpreter stub and replace it with a non-dynamic solution.